### PR TITLE
Create style guide

### PR DESCRIPTION
**Pull request by atjn**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ Tests can be run with:
 yarn test
 ```
 
+## Styleguide
+
+For information about file, class, interface and variable naming conventions, code practices, project file structure and rules about the usage of TS types, read the [styleguide](../styleguide.md) file.
```
Comment (Jesper276) [inline .github/CONTRIBUTING.md:56]:
```suggestion
For information about file, class, interface and variable naming conventions, code practices, project file structure, and rules about the usage of TS types, read the [styleguide](../styleguide.md) file.
```

**Review by Jesper276 — CHANGES_REQUESTED** [review]
LAGTM remember oxford comma

**Review by Jesper276 — DISMISSED** [review]
LGTM

**Review by Jesper276 — APPROVED** [review]
LGTM
GIF

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

**Pull request merged**